packages/react-components/react-carousel-preview/library/src/components/CarouselNavContainer/CarouselNavContainer.types.ts

```diff
@@ -0,0 +1,37 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { CarouselAutoplayButton } from '../CarouselAutoplayButton/CarouselAutoplayButton';
+import { CarouselButtonProps } from '../CarouselButton/CarouselButton.types';
+
+export type CarouselNavContainerSlots = {
+  root: Slot<'div'>;
+  next?: Slot<CarouselButtonProps>;
+  prev?: Slot<CarouselButtonProps>;
+  autoplay?: Slot<typeof CarouselAutoplayButton>;
+};
+
+/**
+ * CarouselNavContainer Props
+ */
+export type CarouselNavContainerProps = ComponentProps<CarouselNavContainerSlots> & {
+  /**
+   * Default: 'inline'
+   * Defines the nav container layout:
+   *
+   * 'inline' - Default controls inline with carousel view
+   *
+   *  inline-wide - Similar to inline but places nav buttons on far left/right
+   *
+   * 'overlay' - Controls overlaid on bottom of carousel viewport,
+   *
+   * 'overlay-wide' - Controls overlaid on bottom of carousel viewport with prev+autoplay/next buttons on far side
+   *
+   * 'overlay-expanded' - Controls overlaid on bottom of carousel viewport, with prev/next buttons on sides vertically centered
+   */
+  layout?: 'inline' | 'inline-wide' | 'overlay' | 'overlay-wide' | 'overlay-expanded';
```

Comment:
I might be missing it - but I couldn't find a story that exercises this component and prop?

Reply:
The story examples use this for the different variants, it's passed in via the CarouselNavContainer